cheatsheets/JSON_Web_Token_Cheat_Sheet_for_Java.md

```diff
@@ -603,10 +603,14 @@ You can also use:
 - [JohnTheRipper](https://github.com/hashcat/hashcat/issues/1057#issuecomment-279651700) to perform the password dictionary attack.
 - [Hashcat](https://hashcat.net/hashcat/).
 
+### Additional Reading
+
+- [JWT Best Practices Internet Draft](https://datatracker.ietf.org/doc/draft-ietf-oauth-jwt-bcp/)
+
 # Authors and Primary Editors
 
 Jim Manico - jim.manico@owasp.org
 
 Dominique Righetto - dominique.righetto@owasp.org
 
-Paul Ionescu - paul.ionescu@owasp.org
\ No newline at end of file
+Paul Ionescu - paul.ionescu@owasp.org
```

Comment:
Kindly remove the author name as we are looking to shift to Github's contributors model.